aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs

```diff
@@ -176,12 +176,12 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [
             max_execution_gas: InternalGas,
             { 7.. => "max_execution_gas" },
-            3_676_000_000,
+            910_000_000, // 91ms of execution at 10k gas per ms
```

Comment:
You need to bump the gas swchedule version as well I think?

Reply:
do I? this is just onchain config change?

we just released to testnet new values of gas schedule, without updating gas version

@vgao1996

Reply:
The version bump is only needed if you are 
(1) adding or removing gas parameters (renaming counts as removing) 
(2) adding breaking changes gated by the feature version